src/test/rpc/Tx_test.cpp

```diff
@@ -528,6 +547,29 @@ class Tx_test : public beast::unit_test::suite
                 client.status.ok(), client.reply);
         };
 
+        auto grpcAccountTx = [&grpcPort](
+                                 uint256 const& id,
+                                 auto binary,
+                                 AccountID const& account) {
```

Comment:
I'd put a return type on the lambda. You should be able leave off the long return types in the body if you do and it makes it easier to see the return type.